src/Core/Makefile.am

```diff
@@ -7,3 +13,10 @@ SUBDIRS = \
 		FSpot.Query \
 		FSpot.Gui \
 		FSpot.Platform
+
+RESOURCES = \
+		FSpot.Core.UnitTest/TestData/taglib-sample.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.xmp \
+		FSpot.Core.UnitTest/TestData/taglib-sample.jpg \
+		FSpot.Core.UnitTest/TestData/taglib-sample-broken.jpg
+		
```

Comment:
That's a good point. I'd really like to increase the number of unit tests that f-spot has. Might be a good project to get all of the tests cleaned up, improve code coverage, and added to the tarball.